test/parallel/test-child-process-cwd.js

```diff
@@ -52,7 +52,14 @@ function testCwd(options, expectPidType, expectCode = 0, expectData) {
   });
 
   child.on('close', common.mustCall(function() {
-    expectData && assert.strictEqual(data.trim(), expectData);
+    if (expectData) {
+      // In Windows, compare without considering case
+      if (process.platform === 'win32') {
```

Comment:
```suggestion
      if (common.isWindows) {
```

Feel free to ignore.

Reply:
Thank you for your advice. I agree that using common.iswindow would be better to maintain the code style. I will work harder to keep the code style and contribute more. By the way, I added you as a co-author, but I'm not sure if it worked. It's my first time contributing, so I'm a bit confused.